<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupWork</class>
 <widget class="QMainWindow" name="GroupWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Учебные группы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="11" column="0" colspan="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Выбрать группу</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="renameGroupBut">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Изменит название группы</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="3">
       <widget class="QPushButton" name="removeGroupBut">
        <property name="text">
         <string>Удалить группу</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="addGroupBut">
        <property name="text">
         <string>Добавить группу</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="Line" name="line_7">
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="addGroup"/>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="newNameGroup"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Выбрать группу</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QComboBox" name="listFac"/>
      </item>
      <item row="6" column="1" rowspan="4">
       <widget class="Line" name="line_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2">
       <widget class="Line" name="line_5">
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="renameGroup"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="listGroupBut_2">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Удалить предмет группе</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="listObject_2"/>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="listObject"/>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Менеджер работы с учебными группами</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QComboBox" name="listGroup"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Выбрать группу</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Выбрать факультет</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="3">
       <widget class="QComboBox" name="removeGroup"/>
      </item>
      <item row="14" column="0" colspan="3">
       <widget class="QLabel" name="statusGroup">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="Line" name="line_6">
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="listGroupBut">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Добавить предмет группе</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>Вернуться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
